main.go

```diff
@@ -66,14 +65,8 @@ func main() {
 		return
 	}
 
-	if config.PageOSName() == config.OSName() {
-		if err = prompter.Run(cmd); err != nil {
-			fmt.Printf("%s\n", err.Error())
-			return
-		}
-	} else {
-		cyan := color.New(color.FgCyan)
-		fmt.Printf("Full command: %s\n", cyan.Sprint(cmd))
+	if err = prompter.Run(cmd); err != nil {
```

Comment:
I suggest not run command while searching pages for different os which may cause errors

Reply:
The config method looks good to me.

Reply:
You are right, but better solution would be if we check the binary is available. It will be a better solution which will cover all errors.

For example; we can check if the binary is present via `which $BINARY_NAME` but it needs to be throughout tested.

Reply:
event the binary exist, we should never run command under wrong os.

For example; `netstat` have different options under linux and macosx.

Reply:
I will work on a solution on this for the next iteration. For now, we can deploy OS setting.